binder/Utils/XamarinAndroid.cs

```diff
@@ -144,5 +145,27 @@ public static string JavaSdkPath
         {
             get { return javaSdkPath.Value; }
         }
+
+        static Lazy<string> msBuildPath = new Lazy<string>(() =>
+        {
+            if (Platform.IsMacOS)
+                return "/Library/Frameworks/Mono.framework/Versions/Current/Commands/msbuild";
+
+            if (Platform.IsWindows)
+            {
+                using (var registryKey = Registry.LocalMachine.OpenSubKey(@"SOFTWARE\Microsoft\MSBuild\ToolsVersions\4.0"))
```

Comment:
You can use this helper code from CppSharp for this: https://github.com/mono/CppSharp/blob/master/src/Core/Toolchains/MSVCToolchain.cs#L357